sechub-pds-solutions/pds-base/docker/PDS-Debian.dockerfile

```
@@ -176,6 +177,9 @@ COPY run.sh /run.sh
 # Copy the additional "hook" script into the container
 COPY run_additional.sh /run_additional.sh
 
+# Copy run script into the container
+COPY /helper "$HELPER_FOLDER"
```

This would copy the absolute path. I'd guess we want the relative path:
```suggestion
COPY helper/ "$HELPER_FOLDER"
```

---

Thanks